### DISCUSSION TITLE
Welcome to Element Discussions!

### DISCUSSION THREAD

**Discussion by kittykat** [element-hq/element-meta#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions to talk about new features in Element. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Randywfl** [comment]
Just wanted to comment:
Two things I don't like about Element:
  * cannot see who is online/active, or any such status.
  * If you sign out they make it a total PITA to sign back in. There is no need to be so anal about signing in to your account, requiring a 2nd device.  Totally anal.  I give up on Element and have to look elsewhere.  Was hoping it could replace Skype, but it's just lame.

**Reply by awtj8o81ryywg793** [reply]
Let me chime in here. I just want to clarify a few things.

> cannot see who is online/active, or any such status.

Matrix itself does support presence, but a homeserver may disable such functionality since the current implementation isn't the best. I do believe that this is either currently being worked on or at least will be at some point. I'm pretty sure that the matrix.org homeserver has such functionality disabled. If Element X doesn't yet support such functionality, I assume it will once the underlying function has been improved. I cannot see such an option within Element X, but I'm not sure if that's because my homeserver has presence disabled or not.

> If you sign out they make it a total PITA to sign back in. There is no need to be so anal about signing in to your account, requiring a 2nd device. Totally anal. I give up on Element and have to look elsewhere. Was hoping it could replace Skype, but it's just lame.

If you sign out of all your devices, which seems to be what happened here (correct me if I'm wrong), others won't be able to encrypt or sign for you. There is ongoing work to support dehydrated devices, enabling you to log out of all your devices while still allowing other users to encrypt and sign for you. As part of the Invisible Crypto initiative, unverified devices will no longer be permitted, requiring you to verify each of your devices. While this may seem awkward, the initiative includes solutions designed to make the process seamless and user-friendly.

You may find the re-run of the "Invisible Crypto" talk from the 2024 Matrix Conference of interest. Here's a [link](https://www.youtube.com/watch?v=axBRh3YUGBM). You can also find the talk's slides [here](https://2024.matrix.org/documents/talk_slides/LABG%202024-09-20%2014_15%20Valere%20Fedronic%20-%20Invisible%20Cryptography.pdf), too.

**Reply by Randywfl** [reply]
Never mind.  Go ahead and cripple it with all the overly anal security ploys you can dream up.  I've had our whole team move onto Wire.  Plenty Secure and FAR LESS problematic.

**Reply by awtj8o81ryywg793** [reply]
Quoted from a [previous reply](https://github.com/element-hq/element-meta/discussions/66#discussioncomment-15020455):

> Never mind. Go ahead and cripple it with all the overly anal security ploys you can dream up. I've had our whole team move onto Wire. Plenty Secure and FAR LESS problematic.

For the record, I'm not employed by Element, nor am I a maintainer for any of their projects. However, I have contributed by opening issues, participating in discussions, and offering community support for some of those projects.

As mentioned in my [last reply](https://github.com/element-hq/element-meta/discussions/66#discussioncomment-14718080), there is ongoing work to address some of the potential annoyances related to verification, end-to-end encryption, security, and other aspects within Matrix. Please refer to the materials I linked to, if you haven't already.

I wish you all the best in your choice of communication platform.

**Reply by Randywfl** [reply]
Too little, too late.  But hopefully ongoing will eventually get a usable platform others can use.  Thanks Jack.

**Comment by elkiaer-dk** [comment]
First, where do I Verify myself and when I have done that, where do I Verify my Verification. 

Above is not a real questions, more a statement that ordinary people (means NOT nerdy IT-people) will not be using Element, when you have to verify all the time. Sometime they can not see where they are going to verify and sometimes they do not know, that they have to verify them self.

In the documentation I think it is very hard to find what is needed so Element can be understood.
But anyway, in the non profit organization which I am a part of, I will try give Element a change by making a guidance on how to use Element and what if something goes wrong. This guidance must be in very short terms otherwise I will loose the members.

Try to look at Sailtrix, but - unfortunately not many do have SailfishOS, which use the swipe technology, that makes it very understandable and easy. And sadly Sailtrix can not use video calls.

**Comment by Randywfl** [comment]
There is no "what if something goes wrong." ...  Use element and something 
*WILL* go wrong and forget about any recovery, which is why we moved to "Wire' instead..
 
 
Thanks,

Randy Walker


----- Original Message -----
From: elkiaer-dk ***@***.***>
Reply-To: element-hq/element-meta ***@***.***>
To: element-hq/element-meta ***@***.***>
Cc: RandyWFL ***@***.***>, Comment ***@***.***>
Sent: 1/28/2026 11:59:37 AM
Subject: Re: [element-hq/element-meta] Welcome to Element Discussions! (Discussion #66)



First, where do I Verify myself and when I have done that, where do I Verify my Verification.
Above is not a real questions, more a statement that ordinary people (means NOT nerdy IT-people) will not be using Element, when you have to verify all the time. Sometime they can not see where they are going to verify and sometimes they do not know, that they have to verify them self.
In the documentation I think it is very hard to find what is needed so Element can be understood.
But anyway, in the non profit organization which I am a part of, I will try give Element a change by making a guidance on how to use Element and what if something goes wrong. This guidance must be in very short terms otherwise I will loose the members.
Try to look at Sailtrix, but - unfortunately not many do have SailfishOS, which use the swipe technology, that makes it very understandable and easy. And sadly Sailtrix can not use video calls.

**Reply by elkiaer-dk** [reply]
There were 3 words in my text that tells you that I can not use Wire "non profit organization".

**Comment by Randywfl** [comment]
I understand.  Nothing worse than a non profit organization. 
 
 
Thanks,

Randy Walker





----- Original Message -----
From: elkiaer-dk ***@***.***>
Reply-To: element-hq/element-meta 
***@***.***>
To: element-hq/element-meta ***@***.***>
Cc: RandyWFL ***@***.***>, Comment ***@***.***>
Sent: 1/29/2026 12:44:50 AM
Subject: Re: [element-hq/element-meta] Welcome to Element Discussions! 
(Discussion #66)



There were 3 words in my text that tells you that I can not use Wire 
"non profit organization".
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: <element-hq/element-
***@***.***>
